lib/components/RunFrame.tsx

```diff
@@ -39,17 +39,29 @@ interface Props {
 
 export const RunFrame = (props: Props) => {
   const [circuitJson, setCircuitJson] = useState<any>(null)
+  const [error, setError] = useState<{
+    phase?: string
+    componentDisplayName?: string
+    error?: string
+    stack?: string
+  } | null>(null)
 
   useEffect(() => {
     async function runWorker() {
       const worker = await createCircuitWebWorker({
         webWorkerUrl: evalWebWorkerBlobUrl,
         verbose: true,
       })
-      const $finished = worker.executeWithFsMap({
-        entrypoint: props.entrypoint,
-        fsMap: props.fsMap,
-      })
+      const $finished = worker
+        .executeWithFsMap({
+          entrypoint: props.entrypoint,
+          fsMap: props.fsMap,
+        })
+        .catch((e) => {
+          // removing the prefix "Eval compiled js error for "./main.tsx":"
+          const message = e.message.split(":")[1]
+          setError({ error: message, stack: e.stack })
```

Comment:
the error is no longer logging on line 70 due to this change, so remove line 70 and add the console.error so that way the stack isn't lost (we can open the console)